package.json

```diff
@@ -20,20 +20,14 @@
   "bugs": "https://github.com/nosir/cleave.js/issues",
   "homepage": "https://github.com/nosir/cleave.js",
   "docs": "https://github.com/nosir/cleave.js/blob/master/README.md",
-  "main": "src/Cleave.js",
+  "main": "dist/cleave.js",
```

Comment:
Can we not change this one?
The vanilla JS component doesn't need any ES6 stuff, so won't have problem when it is built.